response_operations_ui/controllers/message_controllers.py

```diff
@@ -53,19 +69,37 @@ def get_conversation_count(survey_id, conversation_tab, business_id):
         raise ApiError(response)
 
     logger.info("Count successful")
+
     try:
-        return response.json()['total']
+        totals = response.json()['totals']
+
+        # Secure Message uses different identifiers to the tab names used in the ui, this translates the names
+        totals['initial'] = totals.pop('new_respondent_conversations')
```

Comment:
I think this might need to change.  You currently get the totals from the response,  remove new_respondent_conversations and my_conversations and add 'my messages', initital and current to it then return it.
What happens if total contains extra stuff (because we made a future change)?  This can be written in a way so that we can be more specific about what we want (unless we really do want everything, but I'm not so sure about implicitly taking everything)

It could look like 
```python
totals = response.json()['totals']
#not sure about the variable name >_<
ui_totals['initial'] = totals.get('new_respondent_conversations')
ui_totals['my messages'] = totals.get('my_conversations')
ui_totals['current'] = totals[conversation_tab]
```

Reply:
get leaves them in place with the incorrect name, pop replaces them . But there is a small  bug in that we will end up with initial and my messages being null in the case of getting a count for 1 tab. fiixing that.